docker-app/qfieldcloud/settings_utils.py

```diff
@@ -14,9 +14,15 @@ class S3StorageOptions(TypedDict):
     endpoint_url: str
 
 
+class WebDAVStorageOptions(TypedDict):
```

Comment:
```suggestion
class WebDavStorageOptions(TypedDict):
```

Usually we kill the capitals from abbreviations.